package.json

```diff
@@ -62,15 +62,32 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "Slice",
+      "title": "Slice Settings",
```

Comment:
I would leave this as Slice. It's what show's up in the Vscode list. Prepending settings doesn't seem common.
<img width="279" alt="Screenshot 2024-01-11 at 9 59 45 AM" src="https://github.com/zeroc-ice/vscode-slice/assets/2126835/60ed0072-c565-4535-b8c9-feadb77ce21e">